### ISSUE TITLE
Better UX please

### ISSUE THREAD

**Issue by nhrones** [nestdotland/nest.land#295]
**Is your feature request related to a problem? If so, please describe.**
When a user clicks a module, she is redirected.  When using the back button to return to the main page, she is always taken to the top of the main page.  This is very frustrating.

**Describe the solution you'd like**
Right clicking on a module should show a context menu that would allow  an 'Open in new tab' option.
This would allow the user to go back to the main page tab and continue browsing from where she left off.

**Comment by t8** [comment]
Hi @nhrones! Thanks for the feedback. 

If I'm understanding correctly, you'd like to see:

1. A "scroll" rememberer that can save your last scroll position when navigating back and forth from the modules gallery
2. An anchor element on the gallery cards so you can right click and see "open in a new tab"

Let me know if I got this correct. These would both be great hacktoberfest issues!

**Comment by nhrones** [comment]
Yes! You are correct.  There are not that many elements in your database
now, but eventually there could be tens of thousands.
I love the tech-base this service is built on, but the UX is just too
frustrating to use.  I'd also like to see some basic filtering and search
features.
I'm a retired programmer with lots of free time. Let me know how I can help.

**Comment by jasonappah** [comment]
I might be wrong, but according to [this StackOverflow answer](https://stackoverflow.com/a/51057029/5272190), we should just be able to wrap the content inside of the `<router-link>` component in an `<a>` tag. I can try this out and make a PR.

**Comment by t8** [comment]
That would be awesome. Thanks, @jasonappah!

**Comment by t8** [comment]
Oops, we still need a "scroll position rememberer."
